micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -90,7 +90,7 @@ public MnProcessEngineConfiguration(SynchronousTransactionManager<Connection> tr
     public ProcessEngine buildProcessEngine() {
         return transactionManager.executeWrite(
                 transactionStatus -> {
-                    log.info("Building process engine connected to {}", dataSource.getConnection().getMetaData().getURL());
```

Comment:
This will not work. We're not using a StandaloneProcessEngine (which uses getJdbcUrl) but one which uses a managed datasource, i.e. the one provided by Micronaut. Therefore your method always returns "jdbc:h2:tcp://localhost/activiti" unrelated to the configured data source.

I think you need to inject `BasicJdbcConfiguration` and call `getUrl()` (I'm not sure what the difference is to `getConfiguredUrl()`.

Reply:
ok understood